broadinstitute/single_cell_portal_core
# Consolidating feature flag editing into FeatureFlagOptionsController (SCP-3834)

Since #1208 was merged, updating all feature flag values has been moved to the `FeatureFlagOptionsController`.  However, there was still a legacy form available from the admin control panel in the "Edit User Roles" view.  This update removes the ability to edit feature flags from that view, and consolidates into the new forms/controllers added in #1208.

In its place, a convenience button to redirect to the correct form to edit user-based feature flags has been added.

MANUAL TESTING
1. Boot as normal and sign in with an administrator account
2. Go into the admin control panel, click the "User Roles" tab, and then click "Edit" for your user account
3. Confirm that the feature flags form is replaced by a button to redirect to the correct form/controller
![Screen Shot 2021-11-02 at 2 13 23 PM](https://user-images.githubusercontent.com/729968/139922501-00d6775f-7721-4bc0-9f6f-a62e088d4ba1.png)
4. Click the button, and ensure you are redirected to the correct place:
![Screen Shot 2021-11-02 at 2 19 31 PM](https://user-images.githubusercontent.com/729968/139922667-4ec7fc91-1334-42a2-a6c2-d857f8b4f9da.png)

This PR satisfies SCP-3834.
